pkg/types/defaults/installconfig.go

```diff
@@ -82,6 +84,10 @@ func SetInstallConfigDefaults(c *types.InstallConfig) {
 		}
 	}
 
+	if c.Capabilities != nil && c.Capabilities.BaselineCapabilitySet == "" {
+		c.Capabilities.BaselineCapabilitySet = configv1.ClusterVersionCapabilitySetCurrent
```

Comment:
This puts the installer on the hook for updating as the API-declared default changes.  Doesn't seem like a high risk, but I'm also not clear on the benefit vs. just leaving the value unset so the CVO can default it.  For precedent, see #4112 where I removed the need for the installer to care about a property that the CVO can default on its own.

Reply:
+1. That makes sense. I'll remove this here.